__tests__/app.test.js

```diff
@@ -138,15 +138,146 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/100000")
       .expect(404)
       .then(({ body }) => {
-        expect(body.message).toBe("article not found");
+        expect(body.message).toBe("Article not found");
       });
   });
   it("responds with status 400 for invalid non numeric id", () => {
     return request(app)
       .get("/api/articles/banana")
       .expect(400)
       .then(({ body }) => {
-        expect(body.msg).toBe("invalid id");
+        expect(body.message).toBe("Invalid ID");
+      });
+  });
+});
+
+describe("GET /api/articles/:article_id/comments", () => {
+  it("responds with comments array and status 200", () => {
+    return request(app)
+      .get("/api/articles/6/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments).toBeInstanceOf(Array);
+      });
+  });
+  it("responds with all comments, if there are comments", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(11);
+      });
+  });
+  it("responds with status 200 and message for no comments", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.message).toBe("No comments");
+        expect(body.comments.length).toBe(0);
+      });
+  });
+  it("responds with status 200 and array contains comment objects with correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        body.comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe("number");
+          expect(typeof comment.votes).toBe("number");
+          expect(typeof comment.created_at).toBe("string");
+          expect(typeof comment.author).toBe("string");
+          expect(typeof comment.body).toBe("string");
+        });
+      });
+  });
+  it("responds with correct comment data", () => {
+    return request(app)
+      .get("/api/articles/6/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const comment = body.comments[0];
+        expect(comment.body).toBe("This is a bad article name");
+        expect(comment.votes).toBe(1);
+        expect(comment.author).toBe("butter_bridge");
+      });
+  });

```

Comment:
This assertion doesnt need to be here as the test above this already checks all comments for their data type and would work even if the test data changes, but this test would fail if that were to happen.